deploy-agent/deployd/client/serverless_client.py

```diff
@@ -26,7 +26,7 @@
 
 log: logging.Logger = logging.getLogger(__name__)
 
-_DEPLOY_STAGE_TRANSITIONS = dict([(i, i+1) for i in range(DeployStage.PRE_DOWNLOAD, DeployStage.SERVING_BUILD)])
+_DEPLOY_STAGE_TRANSITIONS = dict([(i, i+1) for i in range(DeployStage.PRE_DOWNLOAD.value, DeployStage.SERVING_BUILD.value)])
```

Comment:
it's possible to directly use a dict comprehension here, rather than cast a list comprehension into a dict

Reply:
Addressed and tested in latest push